vulture/core.py

```diff
@@ -105,6 +110,7 @@ def __init__(self, name, typ, filename, lineno, size=1, message=''):
         self.lineno = lineno
         self.size = size
         self.message = message or "unused {typ} '{name}'".format(**locals())
+        self.confidence = CONFIDENCE_MAP.get(typ, 0.6)
```

Comment:
We should pass the confidence values to Item, self._define and self._define_variable, since we'll need different values for defined variables. Function arguments get 100%, other variables get 60%.